src/client/renderingengine.cpp

```diff
@@ -567,6 +568,7 @@ void RenderingEngine::draw_anaglyph_3d_mode(Camera *camera, bool show_hud, Hud *
 			irr::scene::ESNRP_TRANSPARENT +
 			irr::scene::ESNRP_TRANSPARENT_EFFECT + irr::scene::ESNRP_SHADOW;
 	camera->getCameraNode()->setPosition(rightEye);
```

Comment:
if you see more repeated code before those 3 calls, you can propose yes. I prefer doing this in this pr